idaes_examples/mod/hda/hda_ideal_VLE.py

```diff
@@ -397,13 +397,16 @@ def define_metadata(cls, obj):
              'temperature_dew': {'method': '_temperature_dew'},
              'pressure_bubble': {'method': '_pressure_bubble'},
              'pressure_dew': {'method': '_pressure_dew'},
-             'fug_vap_comp': {'method': '_fug_vap_comp'},
-             'fug_liq_comp': {'method': '_fug_liq_comp'},
              })
 
         obj.define_custom_properties(
-            {'dh_vap': {'method': '_dh_vap', "units": obj.derived_units.ENERGY_MOLE},
-             'ds_vap': {'method': '_ds_vap', "units": obj.derived_units.ENERGY_MASS}})
+            {
+                'dh_vap': {'method': '_dh_vap', "units": obj.derived_units.ENERGY_MOLE},
+                'ds_vap': {'method': '_ds_vap', "units": obj.derived_units.ENERGY_MASS},
+                'fug_vap_comp': {'method': '_fug_vap_comp', "units": obj.derived_units.PRESSURE},
```

Comment:
This has me wondering why it wasn't just `fug_phase_comp` to begin with - there are only two phases present. I suspect ti would not take much effort to change over to that either.

Reply:
Given that this is one of the main tutorials, I think we should do this right and make the names follow the standards.

Reply:
I can update the model, as long as there is not some documentation or tutorials that also need updated that's no problem.

Reply:
Should be done.  I ran one of the test notebooks and it seems to work.  We'll see.